lib/OpenQA/Worker/Cache.pm

```diff
@@ -35,173 +37,325 @@ our (@ISA, @EXPORT);
 my $cache;
 my $host;
 my $location;
-my $limit = 50;
-my $db_file;
+our $limit = 50 * 1024 * 1024 * 1024;
```

Comment:
I'm using 50 here, but 100 should be a safe value?... in any case, it could be configured per worker with a minor change on Worker/Engines/isotovideo.pm